File: MunchkinLevelCounter/app/src/main/java/com/gmail/miloszwasacz/munchkinlevelcounter/Player.java
package com.gmail.miloszwasacz.munchkinlevelcounter;

public class Player {

    public String name;
    public int level;

    public Player(String name) {
        this.name = name;
        this.level = 1;
    }

    public Player(String name, int level) {
        this.name = name;
        this.level = level;
    }
}

File: app/src/main/java/com/gmail/miloszwasacz/munchkinlevelcounter/Player.kt
package com.gmail.miloszwasacz.munchkinlevelcounter


class Player(var name: String, value: Int = 1) : BaseItem(value) {
    override fun toString(): String {
        return "Player [name=$name, value=$value, isA=$isA]"
    }
    init {
        isA = "Player"
    }
}